extensions/wikia/PortableInfobox/querypage/AllinfoboxesQueryPage.php

```diff
@@ -104,9 +104,14 @@ public function reallyDoQuery( $limit = false, $offset = false ) {
 			} );
 
 		return array_filter( $result, function ( $tmpl ) {
-			$data = PortableInfoboxDataService::newFromPageID( $tmpl[ 'pageid' ] )->getData();
+			$title = Title::newFromID( $tmpl[ 'pageid' ] );
+			if ( $title && $title->exists() && !$title->isSubpage() ) {
+				$data = PortableInfoboxDataService::newFromTitle( $title )->getData();
```

Comment:
Based on our offline conversation about use cases (doc / Draft / real hierarchy) what do you think: should we eliminate all subpages, or for now just the doc and draft ones?